sdk/src/Core/Amazon.Runtime/DefaultConfiguration.cs

```diff
@@ -81,7 +81,26 @@ public interface IDefaultConfiguration
         TimeSpan? HttpRequestTimeout { get; }
     }
 
-    /// <inheritdoc cref="IDefaultConfiguration"/>
+    /// <summary>
+    /// A defaults mode determines how certain default configuration options are resolved in the SDK. Based on the provided
+    /// mode, the SDK will vend sensible default values tailored to the specific <see cref="DefaultConfigurationMode"/>.
+    /// <para />
+    /// All options above can be configured by users, and the overridden value will take precedence.
+    /// <para />
+    /// <b>Note:</b> for any mode other than <see cref="DefaultConfigurationMode.Legacy"/>, the vended default values
```

Comment:
For later when we merge this to `v4`, we should remove the `Legacy` references here. The updated docs are in: https://github.com/aws/aws-sdk-net/blob/v4-development/sdk/src/Core/Amazon.Runtime/DefaultConfiguration.cs#L21-L36